flexmeasures/api/v3_0/assets.py

```diff
@@ -70,7 +70,7 @@ class AssetAPI(FlaskView):
                 data_key="include_public", load_default=False
             ),
             "page": fields.Int(
-                required=False, validate=validate.Range(min=1), load_default=1
+                required=False, validate=validate.Range(min=1), load_default=None
```

Comment:
This part prevented backward compatibility with use cases where the response wasn't expecting paginated data, to resolve this I made `page` default to None. (I found out, cause a test(`test_auth_token`) was failing because it was receiving paginated data)

What do you think?

@nhoening @Flix6x

Reply:
True, if we default to no pagination for backward-compatibility at the moment, then we need to default to `None`. Maybe the previous deprecated `default` option didn't really work huh?

Reply:
Now that you mention i think i should test that on other API if the `default` actually works

Reply:
So I tested this, the previous `default` doesn't actually default

Reply:
Okay, so we should update this everywhere

Reply:
yes, I am going to create a new PR for that Right now so this one can go forward, but have changed it for this particular API.